src/applications/vaos/tests/covid-19-vaccine/components/ConfirmationPage.unit.spec.jsx

```diff
@@ -85,9 +84,10 @@ describe('VAOS vaccine flow <ConfirmationPage>', () => {
     expect(screen.baseElement).to.contain.text('Main phone: 307-778-7580');
     expect(screen.getByText(/add to calendar/i)).to.have.tagName('a');
 
-    userEvent.click(screen.getByText(/View your appointments/i));
-    expect(screen.history.push.called).to.be.true;
-    expect(screen.history.push.getCall(0).args[0]).to.equal('/');
+    expect(screen.getByText(/View your appointments/i)).to.have.attribute(
```

Comment:
These tests were breaking due to https://github.com/remix-run/react-router/issues/5362

For the tests that were breaking, we were rendering a page outside of our normal route structure, so the url of the page being tested wasn't the actual url, it was just `/`. This means that some of our links back to the homepage looked to react-router like we were going from `/` to `/`, which now doesn't generate a history push. We were testing the link clicks by looking for a push. 

Instead, I've updated them to just look at the href and not click when we don't care about switching to another page, and looking for a history replace when we do care about the next page.